### PR TITLE
Latest post summary block - Part 1

### LINKED ISSUE #8332: Create "Latest post summary" on Insights
- Implement the "Latest post summary" block on the insights screen. 
- Contains tabs for "Views", "Likes" and "Comments"
- Implement a reusable bar graph component for a time period
- Links to the latest post

### PR DESCRIPTION
**Pull request by planarvoid**
Fixes #8332 
Adds `Latest post summary` block to the new Stats. What's still missing in the block is the Graph. The block has 3 states - an empty state, when there is no post yet published, a state when there is a latest post but it had no interactions yet and the normal state with 3 columns with views, comments and likes
The bottom button doesn't have an action attached yet. 

There will be another PR with the actions and tests and the graph.

To test:
* go to Stats
* you see the Latest post summary block on top of the list

### CODE REVIEW

**Review by 0nko — COMMENTED** [review]
👍. Please see the code comments.

**Comment by planarvoid** [comment]
@0nko everything should be fixed 👍

File: WordPress/src/main/res/layout/stats_block_link_item.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              xmlns:tools="http://schemas.android.com/tools"
+              style="@style/StatsBlockLine"
```
Comment (0nko) [inline WordPress/src/main/res/layout/stats_block_link_item.xml]: The button/link is missing a click feedback
```suggestion
              style="@style/StatsBlockLine"
              android:background="?attr/selectableItemBackground"
```

**Review by 0nko — CHANGES_REQUESTED** [review]
Thanks for the changes. 

There's just one issue with the link: If `StatsBlockLine` used paddings instead of a margin, the selectable background would extend the entire width of a block.

**Comment by planarvoid** [comment]
@0nko thanks for the feedback! I've replaced margin with padding and approved your change 👍

**Review by 0nko — APPROVED** [review]
Looks good!

**Pull request merged**